.github/workflows/ci.yml

```diff
@@ -145,17 +145,17 @@ jobs:
         # GHA if this is not present
         # https://twitter.com/steipete/status/921066430220652544?lang=en
         # https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
-        OBJC_DISABLE_INITIALIZE_FORK_SAFETY: YES
+        OBJC_DISABLE_INITIALIZE_FORK_SAFETY: "YES"
       with:
         command: test
-        args: --all -j 4
+        args: -j 4 --workspace --all-features --examples --benches --bins
```

Comment:
is `-j 4` really needed though?  I think cargo automatically figures out the number of cpu's and uses that.  Or is the intention to use more parallelism then there are CPUs?

Reply:
Heh, welcome to the world of CI runners :) If you want to help things along it might make sense to drop down to -j 2 for a moment.

The reasoning is that most of these (esp. GHA runners) OOM very easily if you hit the tests/builds hard. Hence manually limiting to 4 and now potentially even lower for windows builds. 

This should all go away once we move to our infra.